src/snovault/tests/testing_views.py

```diff
@@ -417,12 +417,12 @@ def testing_retry(context, request):
     from transaction.interfaces import TransientError
 
     model = context.model
-    request._attempt = getattr(request, '_attempt', 0) + 1
+    attempt = request.environ.get('retry.attempt')
```

Comment:
oh this must  be the replacement for tm.attempts.

Reply:
this is spelled wrong, 

```'retry.attempt'```
vs
```retry.attempts = 3```
in the config.  Which is an update in encoded too.

ya know what's weird, the region indexer issue says it's failing do to max retrys.  Wonder if this is related.

Reply:
Well, it only shows up in test code though.